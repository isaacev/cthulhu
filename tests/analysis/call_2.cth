fn plus_one(i: Int) -> Int {
  i + 1
}

#[entry]
fn main () -> () {
  plus_one();
}
