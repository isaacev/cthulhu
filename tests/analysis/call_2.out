
  WRONG ARGUMENT TYPE  -------------------------------------------------------

  Expected argument 1 to have the type `Int`, found type `()` instead:

    : call_2
  7 |   plus_one();
    |   ^^^^^^^^^^

